Failure when changing log level

When changing log level as mentioned in the documentation (https://akhq.io/docs/debug.html#debugging-akhq-performance-issues)
The call failed with a http 500 status

`{"message":"Internal Server Error: Failed to inject value for parameter [loggingSystem] of class: io.micronaut.management.endpoint.loggers.LoggersEndpoint\n\nMessage: Multiple possible bean candidates found: [LogbackLoggingSystem, Log4jLoggingSystem]\nPath Taken: \nnew @i.m.m.e.a.Endpoint(\"endpoints.loggers\") i.m.m.e.l.LoggersEndpoint(ManagedLoggingSystem loggingSystem, LoggersManager<Map<String, Object>> loggersManager)\n\\---> new @i.m.m.e.a.Endpoint(\"endpoints.loggers\") i.m.m.e.l.LoggersEndpoint([ManagedLoggingSystem loggingSystem], LoggersManager<Map<String, Object>> loggersManager)","_links":{"self":{"href":"/loggers/org.akhq","templated":false}},"_embedded":{"stacktrace":[{"message":"io.micronaut.context.exceptions.DependencyInjectionException: Failed to inject value for parameter [loggingSystem] of class: io.micronaut.management.endpoint.loggers.LoggersEndpoint\n\nMessage: Multiple possible bean candidates found: [LogbackLoggingSystem, Log4jLoggingSystem]\nPath Taken: \nnew @i.m.m.e.a.Endpoint(\"endpoints.loggers\") i.m.m.e.l.LoggersEndpoint(ManagedLoggingSystem loggingSystem, LoggersManager<Map<String, Object>> loggersManager)\n\\---> new @i.m.m.e.a.Endpoint(\"endpoints.loggers\") i.m.m.e.l.LoggersEndpoint([ManagedLoggingSystem loggingSystem], LoggersManager<Map<String, Object>> loggersManager)\n\tat io.micronaut.context.AbstractInitializableBeanDefinition.resolveBean(AbstractInitializableBeanDefinition.java:2154)\n\tat io.micronaut.context.AbstractInitializableBeanDefinition.getBeanForConstructorArgument(AbstractInitializableBeanDefinition.java:1367)\n\tat io.micronaut.management.endpoint.loggers.$LoggersEndpoint$Definition.instantiate(Unknown Source)\n\tat io.micronaut.context.DefaultBeanContext.resolveByBeanFactory(DefaultBeanContext.java:2342)\n\tat io.micronaut.context.DefaultBeanContext.createRegistration(DefaultBeanContext.java:3162)\n\tat io.micronaut.context.SingletonScope.getOrCreate(SingletonScope.java:80)\n\tat io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:3012)\n\tat io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2774)\n\tat io.micronaut.context.DefaultBeanContext.getBean(DefaultBeanContext.java:1786)\n\tat io.micronaut.context.DefaultBeanContext$BeanExecutionHandle.getTarget(DefaultBeanContext.java:3975)\n\tat io.micronaut.context.DefaultBeanContext$BeanExecutionHandle.invoke(DefaultBeanContext.java:3997)\n\tat io.micronaut.web.router.AbstractRouteMatch.execute(AbstractRouteMatch.java:274)\n\tat io.micronaut.web.router.DefaultUriRouteMatch.execute(DefaultUriRouteMatch.java:38)\n\tat io.micronaut.http.server.RouteExecutor.executeRouteAndConvertBody(RouteExecutor.java:466)\n\tat io.micronaut.http.server.RouteExecutor.lambda$callRoute$5(RouteExecutor.java:443)\n\tat io.micronaut.core.execution.ExecutionFlow.lambda$async$0(ExecutionFlow.java:92)\n\tat io.micronaut.core.propagation.PropagatedContext.lambda$wrap$3(PropagatedContext.java:232)\n\tat java.base/java.util.concurrent.ThreadPerTaskExecutor$TaskRunner.run(ThreadPerTaskExecutor.java:291)\n\tat java.base/java.lang.VirtualThread.run(VirtualThread.java:456)\nCaused by: io.micronaut.context.exceptions.NonUniqueBeanException: Multiple possible bean candidates found: [LogbackLoggingSystem, Log4jLoggingSystem]\n\tat io.micronaut.context.DefaultBeanContext.findConcreteCandidate(DefaultBeanContext.java:2470)\n\tat io.micronaut.context.DefaultApplicationContext.findConcreteCandidate(DefaultApplicationContext.java:675)\n\tat io.micronaut.context.DefaultBeanContext.lastChanceResolve(DefaultBeanContext.java:3335)\n\tat io.micronaut.context.DefaultBeanContext.pickOneBean(DefaultBeanContext.java:3269)\n\tat io.micronaut.context.DefaultBeanContext.findConcreteCandidateNoCache(DefaultBeanContext.java:3225)\n\tat io.micronaut.context.DefaultBeanContext.findConcreteCandidate(DefaultBeanContext.java:3208)\n\tat io.micronaut.context.DefaultBeanContext.findBeanDefinition(DefaultBeanContext.java:2795)\n\tat io.micronaut.context.DefaultBeanContext.resolveBeanRegistration(DefaultBeanContext.java:2764)\n\tat io.micronaut.context.DefaultBeanContext.getBean(DefaultBeanContext.java:1786)\n\tat io.micronaut.context.AbstractBeanResolutionContext.getBean(AbstractBeanResolutionContext.java:210)\n\tat io.micronaut.context.AbstractInitializableBeanDefinition.resolveBean(AbstractInitializableBeanDefinition.java:2137)\n\t... 18 more\n"}]}}`

--- a/build.gradle
+++ b/build.gradle
@@ -38,6 +38,9 @@ configurations.all {
     exclude group: 'org.apache.logging.log4j', module: 'log4j-slf4j2-impl'
     exclude group: 'org.apache.logging.log4j', module: 'log4j-to-slf4j'
     exclude group: 'org.slf4j', module: 'slf4j-reload4j'
+    exclude group: 'org.apache.logging.log4j', module: 'log4j-core'
+    exclude group: 'org.apache.logging.log4j', module: 'log4j-1.2-api'
+    exclude group: 'io.confluent', module: 'confluent-log4j2-extensions'
     // Exclude legacy org.lz4 provider so we can consistently use the relocated at.yawk.lz4
     exclude group: 'org.lz4', module: 'lz4-java'
     // Explicitly exclude missing Confluent telemetry modules from all configurations

